### PR TITLE
[14.0][MIG] barcode_generator_package: migration to 14.0

### PR DESCRIPTION
**Pull request by Raul-S73**
Standard migration to 14.0.

### CODE REVIEW

File: barcodes_generator_package/tests/test_barcode_generator.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestBarcodeGenerator(TransactionCase):
+    def setUp(self):
```
Comment (rousseldenis) [inline barcodes_generator_package/tests/test_barcode_generator.py]: Maybe changing this and switching to SavepointCase

**Review by rousseldenis — APPROVED** [review]
Code review. Minor comment

File: barcodes_generator_package/tests/test_barcode_generator.py (hunk 1 above)
Comment (rousseldenis) [reply barcodes_generator_package/tests/test_barcode_generator.py]: @Raul-S73 I forgot one part of my sentence. I wanted to say changing also setUp to setUpClass :sweat_smile:

Comment (Raul-S73) [reply barcodes_generator_package/tests/test_barcode_generator.py]: @rousseldenis Changes applied

File: barcodes_generator_package/tests/test_barcode_generator.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import SavepointCase
+
+
+class TestBarcodeGenerator(SavepointCase):
+    def setUpClass(self):
```
Comment (rousseldenis) [inline barcodes_generator_package/tests/test_barcode_generator.py]: Nearly:

```suggestion
    @classmethod
    def setUpClass(cls):
```

Comment (Raul-S73) [reply barcodes_generator_package/tests/test_barcode_generator.py]: @rousseldenis Changed

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by aliciagaarzo — APPROVED** [review]

**Comment by Raul-S73** [comment]
Hello @rousseldenis, could you merge the PR? 
Thanks in advance.

**Review by jguenat — APPROVED** [review]
code review & runboat, looks fine thx

**Pull request merged**